calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -24,7 +24,7 @@ Historically, web application firewalls (WAFs) were deployed at the edge of your
 
 ### About {{prodname}} WAF
 
-WAF is deployed in your cluster along with Envoy DaemonSet. {{prodname}} proxies selected service traffic through Envoy, checking HTTP requests using the industry-standard
+WAF is deployed in your cluster with Envoy as a sidecar container on your pods. {{prodname}} proxies selected service traffic through Envoy, checking HTTP requests using the industry-standard
```

Comment:
There seems to be a difference between the old and new concerning services vs deployments. With sidecars, it looks like you need to annotate deployments rather than services.

Is it fair to guess that we should review all instances of 'service' and consider replacing with 'deployment'?

Reply:
@ctauchen yes that's right. In this case I think the sentence can be 

"WAF is deployed in your cluster with Envoy as a sidecar container on your pods. {{prodname}} proxies traffic for the selected pods through Envoy, checking HTTP requests using the industry-standard..."

Reply:
That's why you are the guy from docs, perfect! :)

Reply:
@peterkellydev @radixo The deployment/service question is more complex than I'd realized, particularly with the WAF doc. We have examples using services, a note talking about how you shouldn't apply WAF to system services, and so on. 

I'm thinking that we may need someone who knows what they're doing here (not me) to look through and mark the s/service/deployment locations in the doc.

By my naive understanding, we shouldn't be talking about services much at all, now that we've switched the WAF delivery mechanism to deployments exclusively. What relevance do services have on this page?